pkg/maps/metricsmap/metricsmap.go

```diff
@@ -51,6 +63,16 @@ func (k *Key) String() string {
 	return fmt.Sprintf("reason:%d dir:%d", k.Reason, k.Dir)
 }
 
+// Direction gets the direction in human readable string format
+func (k *Key) Direction() string {
+	return direction[k.Dir]
```

Comment:
I would do a switch statement here to avoid eventual panic due to out of bound access on invalid map data.

Reply:
The data is filled in at BPF so the values are predefined and should be sane, but I can add this check if you like

Reply:
Please do. We may change the format of the BPF map in the future and it can cause values that we don't expect now.

Reply:
done